pygmt/base_plotting.py

```diff
@@ -179,6 +179,9 @@ def colorbar(self, **kwargs):
 
         Parameters
         ----------
+        frame : str or list
+            Set color bar boundary frame, labels, and axes attributes.
```

Comment:
Similar to your PR #707, we already have a short alias `{B}` for docstrings of `frame`.
```suggestion
        {B}
```

Reply:
@seisman I previously used that alias but then I changed it when I saw that it rendered as "Set map boundary frame and axes attributes." To me, that makes it sound like the frame parameter for the colorbar method has an effect on the overall map frame (like using it in basemap, grdcontour, or grdimage) instead of just on the colorbar.

Reply:
Yes, you are right.